zeebe/qa/util/src/main/java/io/camunda/zeebe/qa/util/cluster/TestStandaloneBroker.java

```diff
@@ -235,4 +235,16 @@ public TestStandaloneBroker withCamundaExporter(final String elasticSearchUrl) {
     withBean("camundaSearchClient", searchClient, SearchClientProperties.class);
     return this;
   }
+
+  public TestStandaloneBroker withRdbmsExporter() {
+    withProperty("camunda.database.type", "rdbms");
+    withProperty("spring.datasource.url", "jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1;MODE=PostgreSQL");
+    withProperty("spring.datasource.driver-class-name", "org.h2.Driver");
+    withProperty("spring.datasource.username", "sa");
+    withProperty("spring.datasource.password", "");
+    withProperty("logging.level.io.camunda.db.rdbms", "DEBUG");
+    withProperty("logging.level.org.mybatis", "DEBUG");
+    withExporter("rdbms", cfg -> cfg.setClassName("-"));
```

Comment:
❓ Is it on purpose that we are not setting the RDBMNS exporter class name?

Reply:
Yes, we don't need it because the exporter will be created as spring bean. (Was introduced with this feature: [#22446](https://github.com/camunda/camunda/issues/22446)